packages/react-components/react-datepicker/src/components/CalendarDayGrid/CalendarGridDayCell.tsx

```diff
@@ -2,12 +2,43 @@ import * as React from 'react';
 import { ArrowDown, ArrowLeft, ArrowRight, ArrowUp, Enter } from '@fluentui/keyboard-keys';
 import { getRTLSafeKey } from '@fluentui/react-utilities';
 import { useFluent_unstable } from '@fluentui/react-shared-contexts';
-import { mergeClasses } from '@griffel/react';
+import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
 import { addDays, addWeeks, compareDates, findAvailableDate, DateRangeType } from '../../utils';
+import { weekCornersClassNames } from './useWeekCornerStyles';
+import { tokens } from '@fluentui/react-theme';
 import type { AvailableDateOptions } from '../../utils';
 import type { DayInfo } from './CalendarDayGrid';
 import type { CalendarGridRowProps } from './CalendarGridRow';
 
+const useCornersDateAndPositionStyles = makeStyles({
+  corners: {
+    [`&.${weekCornersClassNames.topRightCornerDate}`]: {
+      borderTopRightRadius: '2px',
+    },
```

Comment:
Since v8 removes and adds classNames to make the hover style for the corners, we simply transfer this to makeStlyes. We will have to make a selector for the className that's being added and then apply the styles based on that.

This can be revisited to remove the removing and adding classNames with onXEvent callbacks in CalendarGridDayCell. For now this works.